src/redis-cli.c

```diff
@@ -3695,6 +3701,7 @@ static sds clusterManagerNodeGetJSON(clusterManagerNode *node,
         "    \"name\": \"%s\",\n"
         "    \"host\": \"%s\",\n"
         "    \"port\": %d,\n"
+        "    \"bus_port\": %d,\n"
```

Comment:
should we use `cluster_bus_port` or something else?
now we use `cluster_bus_port` in cluster-meet.json

Reply:
I suppose cluster_bus_port makes more sense.

Reply:
updated